src/jit/gentree.h

```diff
@@ -945,9 +946,8 @@ struct GenTree
 #define GTF_INX_REFARR_LAYOUT       0x20000000 // GT_INDEX -- same as GTF_IND_REFARR_LAYOUT
 #define GTF_INX_STRING_LAYOUT       0x40000000 // GT_INDEX -- this uses the special string array layout
 
-#define GTF_IND_NONFAULTING      GTF_SET_FLAGS // GT_IND   -- An indir that cannot fault. GTF_SET_FLAGS is not used on indirs.
+#define GTF_IND_NONFAULTING         0x20000000 // GT_IND   -- An indir that cannot fault.
 #define GTF_IND_VOLATILE            0x40000000 // GT_IND   -- the load or store must use volatile sematics (this is a nop on X86)
```

Comment:
How is that possible? In the case of `GTF_IND_VOLATILE`, there would be an ambiguity with `GTF_INX_STRING_LAYOUT` were that the case. As far as I can tell these flags only apply to `GT_IND`.